### ISSUE TITLE
API reference documentation inaccurate

### ISSUE THREAD

**Issue by samiransen23** [Cascella-Group-UiO/HyMD#192]
1. Several API reference documentations of HyMD have been naïvely copied or left unchanged though the functions are different.
2. Some parameters are part of a list, while others are list of lists. This is not carefully documented, but not sure if there is a strict Python way of doing it that ignores this difference. It's a minor difference, but I'd rather make it explicit.
Example:     `phi : list[pmesh.pm.RealField]` but    `phi_fourier : list[ list[pmesh.pm.ComplexField], 3]` and not `phi_fourier : list[pmesh.pm.ComplexField]`

**Comment by hmcezar** [comment]
Good point, we should definitely check and make the documentation consistent with the actual objects.
